taglib/mpeg/mpegheader.cpp

```diff
@@ -240,10 +244,10 @@ void MPEG::Header::parse(File *file, offset_t offset, bool checkLength)
 
   // Set the sample rate
 
-  static const int sampleRates[3][4] = {
-    { 44100, 48000, 32000, 0 }, // Version 1
-    { 22050, 24000, 16000, 0 }, // Version 2
-    { 11025, 12000, 8000,  0 }  // Version 2.5
+  static constexpr std::array sampleRates {
+    std::array<int, 4> { 44100, 48000, 32000, 0 }, // Version 1
+    std::array<int, 4> { 22050, 24000, 16000, 0 }, // Version 2
+    std::array<int, 4> { 11025, 12000, 8000, 0 }   // Version 2.5
   };
 
```

Comment:
This can be simplified to

```c++
  static constexpr std::array sampleRates {
    std::array { 44100, 48000, 32000, 0 }, // Version 1
    std::array { 22050, 24000, 16000, 0 }, // Version 2
    std::array { 11025, 12000, 8000,  0 }  // Version 2.5
  };
```

Reply:
Not sure CTAD works like that. I’ll test.

Reply:
It worked for me with `Apple clang version 14.0.3 (clang-1403.0.22.14.1)` but I'm not sure about other compilers.

Reply:
Done.